packages/ai/src/generate-object/generate-object.ts

```diff
@@ -421,66 +412,20 @@ Default and recommended: 'auto' (best mode for the model).
         request = generateResult.request ?? {};
         response = generateResult.responseData;
 
-        async function processResult(result: string): Promise<RESULT> {
-          const parseResult = await safeParseJSON({ text: result });
-
-          if (!parseResult.success) {
-            throw new NoObjectGeneratedError({
-              message: 'No object generated: could not parse the response.',
-              cause: parseResult.error,
-              text: result,
-              response,
-              usage,
-              finishReason,
-            });
-          }
-
-          const validationResult = await outputStrategy.validateFinalResult(
-            parseResult.value,
+        let object: RESULT;
+        try {
```

Comment:
try catch can be removed if it only rethrows error

Reply:
Good catch ty!